### ISSUE TITLE
[Bug] Cannot rollback other migration than first, migration not found

### ISSUE THREAD

**Issue by Polfo** [Laravel-Backpack/devtools-issues#32]
# Bug report

### What I did

In Migration, 

### What I expected to happen 

http://127.0.0.1:8000/admin/devtools/migration, cloik rollback on the second row, while the first row is not rolled back. 

### What happened

Error message: `migration not found`.
That should either execute or say: `rollback previous migration first`

### What I've already tried to fix it

Rollback the first row. Then the second row can be rolled back.

### Backpack, Laravel, PHP, DB version

When I run ```php artisan backpack:version``` the output is:

### PHP VERSION:
PHP 8.0.2 (cli) (built: Feb  3 2021 18:36:40) ( ZTS Visual C++ 2019 x64 )
Copyright (c) The PHP Group
Zend Engine v4.0.2, Copyright (c) Zend Technologies
    with Zend OPcache v8.0.2, Copyright (c), by Zend Technologies

### LARAVEL VERSION:
v8.74.0@004ea195012d5132eca07a176a6e587c6a74815e

### BACKPACK VERSION:
4.1.61@4400187391eb5b73672c2b166a990cd737277409

### Devtools
backpack/devtools                    1.0.28

### Operating System and Server Setup

I encountered the problem above on my [Windows 10] machine, and my web server is set up using [php artisan serve].

**Comment by tabacitu** [comment]
I can confirm this is happening for me too - thank you @Polfo . 

![2021-12-07 06 01 49](https://user-images.githubusercontent.com/1032474/144964158-dda492d6-0088-401a-acbd-a56432f2255f.gif)

@promatik - I could _swear_ this was working before. I saw it as a huge benefit of a Migrations GUI  - in the command line it's difficult to choose which migration to rollback. In a table it's not. Please investigate what went wrong - this is a weird one.

**Comment by promatik** [comment]
Fixed with https://github.com/Laravel-Backpack/DevTools/pull/268.

Hi @Polfo, thank you for the feedback, unfortunately, Laravel dropped the support to rollback a single/specific migration, and Taylor doesn't want to bring it back. https://github.com/laravel/framework/pull/39901#issuecomment-988940446.

So we had to remove that feature, we will now have a single `Migrate rollback` button to rollback the last batch os migrations.
